packages/react-components/react-toast/stories/Toast/DismissToast.stories.tsx

```diff
@@ -5,21 +5,23 @@ import { useId, Button } from '@fluentui/react-components';
 export const DismissToast = () => {
   const toasterId = useId('toaster');
   const toastId = useId('example');
+  const [removed, setRemoved] = React.useState(false);
   const { dispatchToast, dismissToast } = useToastController(toasterId);
-  const notify = () =>
+  const notify = () => {
     dispatchToast(
       <Toast>
         <ToastTitle intent="success">This is a toast</ToastTitle>
       </Toast>,
-      { toastId },
+      { toastId, onStatusChange: status => setRemoved(status === 'removed') },
```

Comment:
`onStatusChange` does not follow our pattern for event handlers with `(e, data)`. Please update it to be compliant. If `event` is missing, please use `null` instead.

Reply:
as long as you are OK with it _**always**_ being `null`, that's fine